### PR TITLE
Fixed failing tests after adding back throw (bounce) for sint after sig

### PR DESCRIPTION
**Pull request by Skydev0h**
Fixes failing tests after merging #14

### CODE REVIEW

File: tests/wallet-v5-internal.spec.ts [hunk 1]
```diff
@@ -597,7 +597,7 @@ describe('Wallet V5 sign auth internal', () => {
                 (receipt.transactions[1].description as TransactionDescriptionGeneric)
                     .computePhase as TransactionComputeVm
             ).exitCode
-        ).toEqual(0);
```
Comment (Zack435) [inline tests/wallet-v5-internal.spec.ts:600]: 33

File: tests/wallet-v5-internal.spec.ts [hunk 2]
```diff
@@ -597,7 +597,7 @@ describe('Wallet V5 sign auth internal', () => {
                 (receipt.transactions[1].description as TransactionDescriptionGeneric)
                     .computePhase as TransactionComputeVm
```
Comment (Zack435) [inline tests/wallet-v5-internal.spec.ts:598]: 0.053807902 TON

**Review by Zack435 — COMMENTED** [review]
I'm

**Comment by AAmir1361** [comment]
@Zack435

**Comment by AAmir1361** [comment]
Zack435

**Review by Scutua — APPROVED** [review]

**Pull request merged**